buildenv/jenkins/common/pipeline-functions

```diff
@@ -673,7 +673,9 @@ def setup_pull_request() {
     PARSED_PLATFORMS.each { SHORT ->
         LONG_PLATFORM = SHORT_NAMES["${SHORT}"]
         if (LONG_PLATFORM) {
-            PLATFORMS.add(LONG_PLATFORM)
+            LONG_PLATFORM.tokenize(',').each {
```

Comment:
Use `List.addAll()`: `PLATFORMS.addAll(LONG_PLATFORM.tokenize(','))`

Reply:
Don't think we need the `tokenize` with the `addAll`
```